package.json

```diff
@@ -20,14 +20,18 @@
   },
   "homepage": "https://github.com/ishimwezachee/webpack-exercises#readme",
   "devDependencies": {
-    "css-loader": "^6.4.0",
-    "html-webpack-plugin": "^5.5.0",
-    "style-loader": "^3.3.1",
-    "webpack": "^5.60.0",
```

Comment:
- [ ] I can't run your project locally, this is because you delete these dependencies from your devDependencies please add them again. 😄
    - The dependencies are `css-loader`, `html-webpack-plugin` and `style-loader`, your project needs them in order to work properly 🚀